demos/default/app/app-edge/layout.tsx

```diff
@@ -0,0 +1,11 @@
+'use client'
+
+// TODO-APP: support typing for useSelectedLayoutSegment
```

Comment:
Is there an issue for this?

Reply:
This is a Next.js demo site, not our code